File: 프로그래머스/0/181905. 문자열 뒤집기/문자열 뒤집기.java
class Solution {
    public String solution(String my_string, int s, int e) {
        char[] an = new char[my_string.length()];

        for(int i = 0; i < s; i++){
            an[i] = my_string.charAt(i);
        }

        int count = s;
        for(int i = e; i >= s; i--){
            an[count++] = my_string.charAt(i);
        }

        for(int i = e + 1; i < my_string.length(); i++){
            an[i] = my_string.charAt(i);
        }

        String answer = new String(an);
        return answer;
    }
}

File: 프로그래머스/0/120822. 문자열 뒤집기/문자열 뒤집기.cs
using System;

public class Solution {
    public String solution(String my_string) {
        char [] ch = new char[my_string.Length];
        int count = 0;
        for(int i = my_string.Length-1; i >= 0; i--){
            ch[count] = my_string[i];
            count++;
        }
        String answer = new String(ch);
        return answer;
    }
}